service-deployment/clusterconfig-example.yaml

```diff
@@ -115,6 +115,8 @@ clusterinfo:
     prometheus_port: 9090
     # port for node exporter
     node_exporter_port: 9100
+    # cadvisor port
+    cadvisor_port: 4194
```

Comment:
Why config this as a value.

Reply:
as kubernetes cadvisor default expose prometheus format metrics through this port

Reply:
Because we will use our own ca, can you modify this port?